src/sdft_implementations.jl

```diff
@@ -0,0 +1,54 @@
+export SDFT
+
+## Basic SDFT
+
+"""
+    SDFT(n)
+    SDFT(C, n)
+
+Basic method to compute a Sliding Discrete Fourier Transform of window length `n`,
+through the recursive formula:
+
+\$X_{i+1}[k] = e^{j2{\\pi}k/n} \\cdot (X_{i}[k] + x[i+n] - x[i])\$
+
+The transfer function for the `k`-th bin of this method is:
+
+\$H(z) = \\frac{1 - z^{-n}}{1 - e^{j2{\\pi}k/n} z^{-1}}\$
+
+Use `SDFT(C, n)` to obtain results with the precision of the complex data type `C`.
+`C == ComplexF64` by default.
+
+`SDFT` is a subtype of [`AbstractSDFT`](@ref).
+See the documentation of that type for further details about its usage.
```

Comment:
Don't you think that this reference is sufficient? This way, the single example in `AbstractSDFT`'s docstring wouldn't have to be repeated for new methods.

Reply:
Ah I missed it!